### DISCUSSION TITLE
Problemas con ENTREGAR-PEDIDOS

### DISCUSSION THREAD

**Discussion by Fernanda-Bley** [IIC2133-PUC/2024-1#24] [Tarea 0]
Tengo un problema con Entregar pedidos, mi función logra decetar cuando hay pedidos y cuando no pero no sé porque imprime que no hay pedidos cuando no debería. 
**Input recibido**
[input2-2.txt](https://github.com/IIC2133-PUC/2024-1/files/14629073/input.txt)
**Output Esperado**
[correct-output2-2.txt](https://github.com/IIC2133-PUC/2024-1/files/14629074/correct-output.txt)
*Mi codigo*

Lo que hace mi código es iterar por las naves, elegir el destino del primer pedido como el principal y luego iterar sobre los pedidos, entregando los que son para el mismo planeta. Si hay por lo menos una entrega, sumará 1 al contador de entregas.  
**Mi output**
[output.txt](https://github.com/IIC2133-PUC/2024-1/files/14629078/output.txt)
]
En este aparecen 6 "NO HAY QUE ENTREGAR" lo cual me confundé porque todo el resto sale bien. 

Quedo atenta a su respuesta. 
Fernanda.

**Comment by CataAlamos** [comment]
Hola Fernanda! No puedes subir tu código al foro de discusión! Vamos a asignarte a un ayudante que te pueda ayudar, por mientras me puedes hablar a mi por telegram!

**Reply by Fernanda-Bley** [reply]
Preferiría hablar con mi ayudante asignado.
¿Cuánto tiempo se va a demorar que me contacte?
